crates/ruff_linter/resources/test/fixtures/pyupgrade/UP042.py

```diff
@@ -0,0 +1,13 @@
+from enum import Enum
+
+
+class A(str, Enum): ...
+
+
+class B(Enum, str): ...
+
+
+class D(int, str, Enum): ...
+
+
+class E(str, int, Enum): ...
```

Comment:
(I ran `ruff format` over the fixture. It's not a hard rule, since fixtures are often intentionally unformatted, but it's nice when we don't rely on the formatting anywhere.)